Bottom: 3c31d41f6e2e5731cc2c62d351be38dcc194a8b7
Top:    184009853f40110100e55eab6a9ebba19fab1d36
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-03-23 00:13:09 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index ed10b50..79a914a 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
